--- kooldock/admin/cvs.sh.orig	2007-01-16 16:35:24.000000000 +0100
+++ kooldock/admin/cvs.sh	2007-01-16 16:35:51.000000000 +0100
@@ -68,7 +68,7 @@
     echo "*** KDE requires automake $required_automake_version"
     exit 1
     ;;
-  automake*1.6.* | automake*1.7* | automake*1.8* | automake*1.9*) : ;;
+  automake*1.6.* | automake*1.7* | automake*1.8* | automake*1.9* |automake*1.10*) : ;;
   "" )
     echo "*** AUTOMAKE NOT FOUND!."
     echo "*** KDE requires automake $required_automake_version"
